### PR TITLE
Updated news Oct 25th 2020

### PR DESCRIPTION
**Pull request by OsirisTerje**

### CODE REVIEW

File: _posts/2020-07-12-NUnit3TestAdapter-4.0.0-alpha.1.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# NUnit3TestAdapter version 4.0.0-alpha.1 is released.
+
+This is an early pre-release version.
+
+The code has been rewritten/refactored in order to get some of the more complex issues fixed. It does pass all the automatic tests we have, but there are still more tests we would like to take it through, before we release a beta.
+
+We would really appreciate it if you give this alpha a spin. and report whatever you find back to us.
+
+The major fixes now are **Explicit** runs are fully back, both in Visual Studio and on command line with dotnet test and vstest.console.
```
Comment (mikkelbu) [inline _posts/2020-07-12-NUnit3TestAdapter-4.0.0-alpha.1.md:9]: This sentence sounds odd to me (and I'm actually unsure exactly what we want to communicate)
```suggestion
The major fix in this release is to bring back the **Explicit** test runs, both in Visual Studio and on the command line with dotnet test and vstest.console. In addition to this, the implementation handles even more cases than before.
```

**Review by mikkelbu — APPROVED** [review]
I've have a handful of minor comments/suggestions.

File: _posts/2020-07-12-NUnit3TestAdapter-4.0.0-alpha.1.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# NUnit3TestAdapter version 4.0.0-alpha.1 is released.
+
+This is an early pre-release version.
+
+The code has been rewritten/refactored in order to get some of the more complex issues fixed. It does pass all the automatic tests we have, but there are still more tests we would like to take it through, before we release a beta.
+
+We would really appreciate it if you give this alpha a spin. and report whatever you find back to us.
```
[the remark this answers is not in the document]
Comment (jnm2) [reply _posts/2020-07-12-NUnit3TestAdapter-4.0.0-alpha.1.md:7]: I don't think a comma would go there. Removing the period is good though.

File: _posts/2020-07-11-NUnit3TestAdaåter-3.17.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+NUnit3TestAdapter version 3.17.0 is released.
```
Comment (jnm2) [inline _posts/2020-07-11-NUnit3TestAdaåter-3.17.md:1]: Typo `Adaåter` in the filename

File: _posts/2020-07-12-NUnit3TestAdapter-4.0.0-alpha.1.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+In v4 this has changed to transforming the xml to an object model at the time it is loaded from the engine, and then working on the object model instead. This brings back the **Explicit** feature, and the intention is to pave the way for other changes we can do. Further, under discussion is a possible drop of supporting the vsix variant, and also set the minum supported framework version to 4.5. Due to the internal rewrite there might be a intermittent loss of some features, we got one in the alpha.1 version, which is fixed now for the beta, but there might me more. However, there are around 20000 downloads now, with very few issues reported, so it might be better then feared. The changes will also pave the way for a possible better handling of the FQN issues we have - which are very annoying, and which are near impossible to handle in v3. We still need some more from MS in order to do this, but v4 should then be able to handle it. When the adapter was first written back in 2011/2012 it was a much simpler "thing" than it has become over the years. At that time the XML approach was ok, but now, the code is simply too cluttered to continue.
```
[the remark this answers is not in the document]
Comment (jnm2) [reply _posts/2020-07-12-NUnit3TestAdapter-4.0.0-alpha.1.md:17]: XML and VSIX should be uppercased consistently.

**Review by jnm2 — APPROVED** [review]

**Comment by jnm2** [comment]
@OsirisTerje I was going to submit an entry to the .NET Foundation newsletter about the 0.6 NUnit.Analyzers release. Would you like me to include a condensed version of this too?

**Pull request merged**